crates/stdlib/src/sqlite.rs

```diff
@@ -833,10 +833,11 @@ mod _sqlite {
     #[derive(PyPayload)]
     struct Connection {
         db: PyMutex<Option<Sqlite>>,
-        detect_types: c_int,
+        initialized: PyAtomic<bool>,
+        detect_types: PyAtomic<c_int>,
         isolation_level: PyAtomicRef<Option<PyStr>>,
-        check_same_thread: bool,
-        thread_ident: ThreadId,
+        check_same_thread: PyAtomic<bool>,
+        thread_ident: PyMutex<ThreadId>,
```

Comment:
```suggestion
        thread_ident: PyMutex<ThreadId>,  // TODO: Use atomic
```
ThreadId is representable in u64. But not convenient yet.